composeml/label_times/description.py

```diff
@@ -20,11 +20,9 @@ def describe_label_times(label_times):
     metadata = label_times.settings
     target_column = metadata["label_times"]["target_columns"][0]
     target_type = metadata["label_times"]["target_types"][target_column]
-    target_dataframe_name = metadata["label_times"]["target_dataframe_name"]
 
     settings = {
         "target_column": target_column,
-        "target_dataframe_name": target_dataframe_name,
```

Comment:
I think we may still want to track `target_dataframe_index` in the settings

Reply:
Fixed https://github.com/alteryx/compose/pull/353/commits/ae874cbd2c5eae062b007fb8121e8dc9a792b2b9